website/_i18n/en/user-guides/pactus-testnet/index.md

```diff
@@ -0,0 +1,96 @@
+---
+version: 8
+---
+
+## Preface
+
+Pactus Blockchain has a Test network for developers to test their software which is using Pactus, users
+that are wiling to test Pactus, etc. This testnet is called Phoenix and you can interact with
+phoenix with official Pactus pieces of software that you use for main network. In this guide we are
+going to show how you can use them to connect and interact with Phoenix.
+
+## Download
+
+To get started, download and extract the Pactus software
+[here]({{ site.baseurl }}/download).
+Once you have downloaded and extracted the Pactus software file, you can proceed to the next step.
+
+---
+
+## Phoenix network
+
+Pactus has 3 different genesis block types for mainnet, testnet and localnet (used for setting up
```

Comment:
Pactus offers three distinct genesis block types for mainnet, testnet, and localnet (used for developing a dev environment). The genesis type is defined within your Pactus working directory and wallet file once it has been created or initialized. Subsequent commands will automatically determine the network type based on this information, eliminating the need to specify the network being used.